index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>WEB CREATIONS</title>
+        <link rel="stylesheet" href = "style.css">
+    </head>
+    <body>
+
+    
+        <header>
+            <div class="main">
+                <div class="logo">
+                    <img src="image/logo.png">
+                <ul>
+                    <li><a href ="*">home</a></li>
+                    <li><a href ="*">services</a></li>
+                    <li><a href ="*">gallery</a></li>
+                    <li><a href ="*">about</a></li>
+                    <li><a href ="*">contact</a></li>
+                </ul>
+
+           </div>
+
+           <div class="title">
+               <h1> UBUNI-SHULE WEB CREATIONS</h1>
+           </div>
+
```

Comment:
this space is not necessary

Reply:
which one brother

Reply:
ok i have seen it

Reply:
i have changed it

Reply:
before i merge i want your last sugestion